manager-bundle/src/Resources/config/services.yml

```diff
@@ -48,3 +43,9 @@ services:
         class: Contao\ManagerBundle\Security\Logout\LogoutHandler
         arguments:
             - '@?contao_manager.jwt_manager'
+
+    contao_manager.twig.file_extension_filter_iterator:
```

Comment:
What are the rules here? I thought we'd use FQCN for new definitions?

Reply:
No. FQCN are currently only used for  controllers. (This might change in the future though.)

Reply:
I see.

Reply:
We might want to define such CS rules somewhere... 

(@fritzmg Does a 'contributing' section belong to the docs? Or should this be in the repo?)

Reply:
There is a "Guides" section, however imho that should go into a CONTRIBUTING.md of the contao/contao repo, shouldn't it?

Reply:
> No. FQCN are currently only used for controllers. (This might change in the future though.)

But there also exist FQCN aliases for other services, not just for controllers.